docs/GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -54,194 +24,171 @@ export COBALT_PIPELINE_NAME="Cobalt CICD Pipeline"
 
     * Add the Azure Subscription being used for Cobalt as a *Service Connection*
-        * Return to your Azure DevOps subscription
-        * Find and select the Project Settings tab at the bottom of the screen
-        * Under the Pipelines menu select Service Connections
-        * From the Service Connections menu, select [+New Service Connection]
-        * Choose Azure Resource Manager from the dropdown then a name for your service (ex. Cobalt Deployment Administrator-`<YourTenantName>`). The name should make sense to users and will be directly referenced in pipeline variable groups later.
-        * Use the full version of the service connection dialog in order to enter your service principal credentials (AAD Key, AAD App ID, Tenant, etc.)
+        1. Return to your Azure DevOps subscription
+        2. Find and select the Project Settings tab at the bottom of the screen
+        3. Under the Pipelines menu select Service Connections
+        4. From the Service Connections menu, select [+New Service Connection]
+        5. Choose Azure Resource Manager from the dropdown then a name for your service (ex. Cobalt Deployment Administrator-`<YourTenantName>`). The name should make sense to users and will be directly referenced in pipeline variable groups later.
+        6. Use the full version of the service connection dialog in order to enter your service principal credentials (AAD Key, AAD App ID, Tenant, etc.)
 
             ![Service Connection menu](https://user-images.githubusercontent.com/10041279/63485304-63b59c00-c468-11e9-8e47-721a2e43ecb9.png)
 
-        * Verify and Save the connection
+        7. Verify and Save the connection
         > NOTE: Take note of the custom name given to this service connection. This will be referenced in later steps needed to configure env variable groups.
 
     * Enable multi-stage pipelines
-        * Find your signed-in avatar/image and select preview features from the drop down menu
+        1. Find your signed-in avatar/image and select preview features from the drop down menu
 
-            ![Preview Features menu](https://user-images.githubusercontent.com/10041279/63486065-8eedba80-c46b-11e9-90f0-7f931f909ffa.png)
+            ![Preview Features menu](./images/preview.png)
 
-        * Toggle Multi-stage pipelines
+        2. Toggle Multi-stage pipelines
 
     * Configure *Infrastructure Pipeline Variables* as the first of two variable groups
-        * Select Pipelines tab from within side-navigation menu then select Library tab
-        * Click [+Variable group] and name it "Infrastructure Pipeline Variables"
-        * Add the following variables:
+        1. Select Pipelines tab from within side-navigation menu then select Library tab
+        2. Click [+Variable group] and name it "Infrastructure Pipeline Variables"
+        3. Add the following variables:
 
             | Name   | Value | Var Description |
```

Comment:
Are these variables already described in the AzDO pipeline readme? If yes, then we should include a link to the markdown table.

Reply:
Document now provides a link to a markdown table in the infrastructure readme (https://github.com/microsoft/cobalt/tree/master/devops/providers/azure-devops/templates/infrastructure) rather than listing out a table.